### ISSUE TITLE
fine-tuning

### ISSUE THREAD

**Issue by rabbit218218** [LMD0311/PointMamba#51]
Why can't I achieve the 93.6% metric when fine-tuning using your pretrain.pth on top of the source code?

**Comment by LMD0311** [comment]
As mentioned in https://github.com/LMD0311/PointMamba/issues/16, https://github.com/LMD0311/PointMamba/issues/47, different machines or development environments may cause fluctuations in results. Our experiments are conducted on a single NVIDIA 4090 GPU.

You may also find this reported by other work, here I quote:

>The performance on ModelNet40 of almost all methods are not stable, see (https://github.com/CVMI-Lab/PAConv/issues/9#issuecomment-873371422).
The best way to reproduce the results is to test with a pretrained model for ModelNet40.

You can test the checkpoint provided in the README.md, which should get similar results.
